.github/workflows/julia-concept-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Julia Concept CI
+
+on:
+  push:
+    paths:
+    - 'languages/julia/config.json'
```

Comment:
So this is temporary, right? We want this to work for all languages. Was this for testing purposes only?

Reply:
This is related to https://github.com/exercism/v3/pull/718#discussion_r390363519